### ISSUE TITLE
Custom property for person display name is ignored

### ISSUE THREAD

**Issue by lnhrdt** [PostHog/posthog#14693]
## Bug description

I have a custom property called `displayName` configured as the first preferred person display name:
<img width="683" alt="Screenshot 2023-03-10 at 9 05 28 PM" src="https://user-images.githubusercontent.com/4040241/224466047-57842083-f852-4f3f-9868-a208b839da4e.png">

However when persons are displayed in Live Events, the value of `displayName` is not used. Rather, I see the value of whatever value is in the 2nd priority (I've tried both name and email, they both work).

## Environment

PostHog Cloud US, project ID: 1904

## Additional context

I originally suggested this feature (#9169) but I'm not sure when this stopped working.

**Comment by mariusandra** [comment]
Hey @lnhrdt, I added you to the feature flag / beta of a new page called "Event Explorer" that is a SQL-enhanced version to replace the old "Live Events" page. The custom display names do work there. We will release this shortly (within the week? today?) to all users in any case.

**Comment by lnhrdt** [comment]
@mariusandra I see the proper display name in my account now, thank you!

**Comment by lnhrdt** [comment]
Yep, this was fixed two years ago! 😂 Not sure if it’s standard practice for customers to close issues, but I'll close it now.
